recipes/phinx.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+/**
+ * Phinx recipe for Deployer
+ *
+ * @author    Alexey Boyko <ket4yiit@gmail.com>
+ * @copyright 2016 Alexey Boyko 
+ * @license   MIT https://github.com/deployphp/recipes/blob/master/LICENSE
+ *
+ * @link https://github.com/deployphp/recipes
+ *
+ * @see http://deployer.org
+ * @see https://phinx.org
+ */
+
+//Default configuration
+env(
+    'phinx', [
+        'environment' => '',
+        'date' => '',
+        'configuration' => '', 
+        'target' => '',
+        'seed' => '',
+        'parser' => ''
+    ]
+);
+env('phinx_path', 'getPhinx'); 
```

Comment:
Can you move this function to an inline closure to not pollute the global namespace?